app/assets/stylesheets/content.css.scss

```diff
@@ -29,4 +29,22 @@ p.long-form {
 
 .content-field {
   min-height: 140px;
+}
+
+.formatted-text {
+  p {
+    margin-bottom: 20px !important;
+  }
+
+  ul {
+    list-style: initial;
+
+    li {
+      list-style-type: initial !important;
+    }
+  }
+
+  ol {
+    padding: 0;
+  }
 }
```

Comment:
Missing a newline! Install the [editorconfig](http://editorconfig.org/) plugin into whatever editor you use, then these will be automatically taken care of.